arcade/experimental/__init__.py

```diff
@@ -1,17 +1,23 @@
 """
 Experimental stuff. API may change.
 """
-from .video_player import VideoPlayer
-from .video_player import VideoPlayerView
+from .video_player import VideoPlayer, VideoPlayerView
 from .texture_render_target import RenderTargetTexture
 from .shadertoy import Shadertoy, ShadertoyBuffer, ShadertoyBase
 from .crt_filter import CRTFilter
 from .bloom_filter import BloomFilter
 
+# Keep cv2 an optional dependency
+try:
+    from .video_cv2 import VideoPlayerCV2, CV2PlayerView
+except ImportError:
+    pass
 
 __all__ = [
     "VideoPlayer",
     "VideoPlayerView",
+    "VideoPlayerCV2",
+    "CV2PlayerView",
```

Comment:
@cspotcode Should inclusion of these classes in `__all__` be conditional as well?

Reply:
Updated this in [03fa9dc](https://github.com/pythonarcade/arcade/pull/1786/commits/03fa9dca617fe2bb1ab4ccf86b6ceb0679e79f9b). This does cause the linting test to fail as an unused import though... added an ignore here.

Reply:
@pushfoo 

Why don't we put the conditional around `import cv2` within `video_vc2.py`, and remove the conditional from `__init__.py`?  Easier to reason about, and it conveniently sidesteps this `__all__` issue.

As for `__all__`, 2 factors to consider:

a) static tooling should understand the types of exported items
b) python should behave at runtime in these situations:
```python
import arcade.experimental
from arcade.experimental import * # <-- this requires any __all__ items we can't import to be declared in another way, perhaps `MissingItem = None`
```

Reply:
@pushfoo since einarf and MrWardKKHS already talked about where to put the conditional, keeping it here as-is makes sense.  It prevents a runtime error if anyone does `from arcade.experimental import *`.  Any static tooling issues can be fixed in the future, and we can't detect them without #1791.